packages/core/src/app/ui/loading/LoadingOverlay.tsx

```diff
@@ -21,7 +21,7 @@ const LoadingOverlay: FunctionComponent<LoadingOverlayProps> = ({
                 {unmountContentWhenLoading && isLoading ? null : (
                     <div
                         style={{
-                            display: hideContentWhenLoading && isLoading ? 'none' : undefined,
+                            visibility: hideContentWhenLoading && isLoading ? 'hidden' : undefined,
```

Comment:
Could you please explain the reason of changing this?
This component is everywhere, I understand the difference it will make, I'm afarid a thorough testing of the whole checkout is required.

Reply:
By now it is the only way to make work the changes. Somehow `display: none` affect the the Stripe API appearance classes that are in the DOM, and don't pass through some values. Those values appears as 'NaNpx' so while this issue  I don't know where exactly is coming that bug when we set `Labels: floating` in the Stripe appearance. I thought it could be an option but if is a problem it would be better to find another way to make it work properly.

Reply:
I have tested this change, and found no noticeable visual changes.
Think we can go ahead with this, and please apply changes to `packages/ui/src/loading/LoadingOverlay.tsx`, too.